File: coding_meetup_#13.js
function isLanguageDiverse(list) {
  // first of all we need to get the count of every language
  let languageCount={}
  list.forEach((dev)=>{
    if(dev.language in languageCount)languageCount[dev.language]++
    else languageCount[dev.language]=1
  })
//The language can't be more than 2 times higher than any of the other languages to be regarded as language-diverse.
    console.log(languageCount)
    if((languageCount.Python>2*languageCount.Ruby)||(languageCount.Python>2*languageCount.JavaScript))return false
    else if((languageCount.JavaScript>2*languageCount.Python)||(languageCount.JavaScript>2*languageCount.Ruby))return false
    else if((languageCount.Ruby>2*languageCount.JavaScript)||(languageCount.Ruby>2*languageCount.Python))return false
    else return true
  }

File: coding_meetup_#13.py
def is_language_diverse(lst): 
    javascript_devs=list(filter(lambda dev:dev['language']=="JavaScript" , lst))
    python_devs=list(filter(lambda dev:dev['language']=="Python" , lst))
    ruby_devs=list(filter(lambda dev:dev['language']=="Ruby" , lst))
    if len(python_devs)>2*len(ruby_devs) or len(python_devs)>2*len(javascript_devs):
        return False
    elif len(javascript_devs)>2*len(python_devs) or len(javascript_devs)>2*len(ruby_devs):
        return False
    elif len(ruby_devs)>2*len(javascript_devs) or len(ruby_devs)>2*len(python_devs):
        return False
    else:
        return True